app/controllers/donations_controller.rb

```diff
@@ -64,6 +64,11 @@ def new
   def edit
     @donation = Donation.find(params[:id])
     @donation.line_items.build
+
+    audited_items_ids = Audit.all.collect { |i| i.items.pluck(:id) }.flatten
```

Comment:
This is quite an expensive query to load everytime user visits edit page. Any optimization suggestions?

Reply:
We're only interested in the audits that have been finalized since the donation was created, so you should be able to narrow that down considerably.

Reply:
@cielf added a `finalized` clause.